pkg/sso/client.go

```diff
@@ -74,6 +77,10 @@ type GetAuthorizationURLOptions struct {
 	// The app/company domain without without protocol (eg. example.com).
 	Domain string
 
+	// Authentication service provider descriptor.
+	// Currently only used for ConnectionType.GoogleOAuth
```

Comment:
We are not accessing this with `ConnectionType.GoogleOAuth`. It may mislead the reader.

=> `Provider is currently only used when the connection type is GoogleOAuth.`